File: Chess.DataAccess/Entities/Game.cs
using Chess.DataAccess.Helpers;
using System.Collections.Generic;

namespace Chess.DataAccess.Entities
{
    public class Game : Entity
    {
        public string Fen { get; set; }
        public GameStatus Status { get; set; }
        public virtual ICollection<Move> Moves { get; set; }
        public virtual ICollection<Side> Sides { get; set; }

        public Game()
        {
            Moves = new List<Move>();
            Sides = new List<Side>();
        }

    }
}

File: frontend/src/app/core/models/chess/game.ts
import { Side } from './side';
import { Move } from './move';
import { GameStatus } from './gameStatus';
import { GameSide } from './gameSettings';

export class Game {
    id?: number;
    moves?: Move[];
    status?: GameStatus;
    creationDate?: Date;
    constructor(
        public fen: string,
        public sides: Side[]
        ) {}
}